test-resources/swift/cleanup_rules/input/SampleClass.swift

```diff
@@ -492,5 +492,7 @@ class SampleClass {
          } else {
             toBeDeletedInequality12b()
          }
+
+         x = condition() ? TestEnum.stale_flag_one.isEnabled : true
```

Comment:
I think this is probably fine in practice, but an interesting caveat is that if `condition()` has side-effects, this actually does change program behavior beyond setting the flag value to a constant, since it skips the execution of `condition()`.

Reply:
yes, i see what u mean.
But in my experience folks want such aggressive simplifications. They do not want us to be conservative.